src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -0,0 +1 @@
+Hello World
```

Comment:
This file is not related to this PR. Could you remove it?

Reply:
Argh! Removed